.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
What is the purpose of this env?

Reply:
It is a temporary 'fix' for an error in eslint version :(